src/plugins/charts/bar/pluginSpec.js

```diff
@@ -230,29 +293,36 @@ describe("the plugin", function () {
                             domain: 1
                         }
                     }, {
-                        key: "some-key.foo.name.45",
-                        name: "Some dotful attribute",
+                        key: "some-key",
+                        name: "Some attribute",
+                        formatString: '%0.2f[]',
                         hints: {
                             range: 1
-                        }
+                        },
+                        source: 'some-key'
                     }, {
-                        key: "some-other-key.bar.344.rad",
-                        name: "Another dotful attribute",
+                        key: "some-other-key",
+                        name: "Another attribute",
+                        format: "number[]",
                         hints: {
                             range: 2
-                        }
+                        },
+                        source: 'some-other-key'
                     }]
                 }
             };
 
             const applicableViews = openmct.objectViews.get(barGraphObject, mockObjectPath);
             const plotViewProvider = applicableViews.find((viewProvider) => viewProvider.key === BAR_GRAPH_VIEW);
-            const barGraphView = plotViewProvider.view(testDomainObject, [testDomainObject]);
+            const barGraphView = plotViewProvider.view(barGraphObject, [barGraphObject]);
             barGraphView.show(child, true);
-            expect(testDomainObject.configuration.barStyles.series["test-object"].name).toEqual("Test Object");
             mockComposition.emit('add', dotFullTelemetryObject);
-            expect(testDomainObject.configuration.barStyles.series["someNamespace:~OpenMCT~outer.test-object.foo.bar"].name).toEqual("A Dotful Object");
-            barGraphView.destroy();
+            Vue.nextTick().then(() => {
+                const plotElement = element.querySelector('.cartesianlayer .scatterlayer .trace .lines');
+                expect(plotElement).not.toBeNull();
+                barGraphView.destroy();
+                done();
```

Comment:
No need to call `done()`, you can just return the promise. When deleting `done()` make sure to remove it from the function arguments as well.

Reply:
done ! Get it? huh? huh? :P